code/global/RPCs.cs

```diff
@@ -144,19 +144,19 @@ public static void ClientClosePostRoundMenu()
         [ClientRpc]
         public static void ClientOnPlayerCarriableItemPickup(Entity carriable)
         {
-            Event.Run("tttreborn.player.carriableitem.pickup", carriable as ICarriableItem);
+            Event.Run(TTTEvent.Player.CarriableItem.PickUp, carriable as ICarriableItem);
```

Comment:
Can you switch PickUp to an inventory event instead of a carriable item event?

Reply:
We spoke about this away from GitHub and felt that they fit best in inventory events.

Reply:
Yup, I think `inventory` makes sense here.